smithy-diff/src/main/java/software/amazon/smithy/diff/evaluators/ChangedEnumTrait.java

```diff
@@ -84,7 +84,7 @@ private List<ValidationEvent> validateEnum(ChangedShape<Shape> change, Pair<Enum
                                 .severity(Severity.ERROR)
                                 .message(String.format("Enum value `%s` was removed", definition.getValue()))
                                 .shape(change.getNewShape())
-                                .id(getEventId() + REMOVED)
+                                .id(getEventId() + REMOVED + definition.getValue())
```

Comment:
Enum values support arbitrary content, so this seems problematic to include them in the ID. Ideally these events would use the `name` of an enum definition if available, and if not... I dunno. Maybe the index of the entry? Or a name derived from the value?

Reply:
Yeah sounds good. The caveat I can think of is that what is, say index 2, today won't be index 2 in the future if index 2 is removed. Not sure if that matters for you.

Reply:
Shouldn't be an issue since we only care about these events in the scope of a single Trebuchet feature and not after that